### PR TITLE
Cloud Infrastructure Security for AWS - GA

### PR DESCRIPTION
**Pull request by jpipkin1**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request makes the following article ready for GA:
https://help.sumologic.com/docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws/

It also adds a release note.

Issue: Asana ticket "[Documentation for Cloud Infrastructure Security for AWS - GA](https://app.asana.com/0/1183741177809740/1206048684765301/f)"

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Comment by jpipkin1** [comment]
This PR is in draft.

File: blog-service/2024-01-16-aws-for-cis.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+This feature was [previously only available in a preview form](/release-notes-service/2023/11/27/cis-for-aws/). It is now available for general use.
+
+Here are some of the key features the new solution offers: 
+* **Risk overview**. See a summary of all resources that pose risks in a single dashboard that rolls up the findings from other dashboards. Get an action plan for addressing the most important areas of concern.
+* **Active threats**. See active threats in APIs, resources, and storage.
+* **Misconfigurations**. See areas in your environment that need to be addressed because they fail best practice security controls.
+* **Suspicious activity assessment**. See suspicious activity across users, web interactions, networks, and Identity Access Management (IAM).
+
+To learn how you can set up and use Cloud Infrastructure Security for AWS, check out our technical documentation [here](/docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws).
```
Comment (kimsauce) [inline blog-service/2024-01-16-aws-for-cis.md:24]:
```suggestion
To learn how you can set up and use Cloud Infrastructure Security for AWS, check out our [technical documentation](/docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws). 

```

File: blog-service/2024-01-16-aws-for-cis.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Here are some of the key features the new solution offers: 
+* **Risk overview**. See a summary of all resources that pose risks in a single dashboard that rolls up the findings from other dashboards. Get an action plan for addressing the most important areas of concern.
+* **Active threats**. See active threats in APIs, resources, and storage.
+* **Misconfigurations**. See areas in your environment that need to be addressed because they fail best practice security controls.
+* **Suspicious activity assessment**. See suspicious activity across users, web interactions, networks, and Identity Access Management (IAM).
+
+To learn how you can set up and use Cloud Infrastructure Security for AWS, check out our technical documentation [here](/docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws). 
+
+:::note
```
Comment (kimsauce) [inline blog-service/2024-01-16-aws-for-cis.md:26]:
```suggestion
:::note Action Required
```

**Review by kimsauce — APPROVED** [review]
Approved with a couple opt suggestions

**Review by kimsauce — APPROVED** [review]

**Pull request merged**